content/blog/global-case-study.md

```diff
@@ -16,76 +16,82 @@ A leading comprehensive global mental health solution for employers. It started
 
-### Tech Debt
-
-In the JavaScript world, it doesn't take long for dependencies to become stale. While relatively older versions of most of the major libraries are stable, we cannot leverage the new and improved features that the newer versions provide to help us complete our job easily. In certain cases, it was mandatory to reach to a minimum version requirement to be able to add internationalization libraries.  We learnt this the hard way when we were initially trying to add internationalization changes to the codebase. Thus, as our first step, we identified the necessary libraries that required upgrades, resulting into a chain of few other dependent libraries as well, and spent few sprints solely to take care of it. This also led us to recognize several other dependencies that we could benefit from upgrading and helped plan the process to tackle this later on, which deserves its own separate story.
-
-### Internationalization
+### UI Translations
 
-#### Translations
+**Challenge**: Translating the UI based on user's language preference
 
-We had translatable content stored both on the frontend and backend, each of which had their own challenges.
+**Solution**:
 
-#### Frontend
+- Identifying and extracting string literals to JSON locale files
+- Update existing usages of string literals with localized versions
+- Introduce third-party translation provider Crowdin
+- Integrate Crowdin with GitHub PR workflow to provide automated translations
 
-A simple local frontend application will have all the static content in its codebase itself and ours was no different. Once the configuration for internationalization were in place, the next step was to actually move all the static string literals to a specific file format. We chose a standard file format - JSON to store all our translations and thus when moved, the static content got replaced with references to the content in these JSON files. This makes it easier to load the translations on switching locales.
+**Outcome**: Users can use the application in their preferred language.
 
-Since the content is static, the translations need to be generated and stored before going live. Comes into picture, Crowdin, a third-party localization service. The service syncs up with the repository on GitHub and reads only the locales folder that it needs to translate. Once the main branch is translated, we need to ensure that all the subsequent features created by developers adhere to the same standard format so that Crowdin can easily keep translating any new content. To ensure this, Crowdin was setup in such a way that it creates a Pull Request with translations pointing to the relevant feature branch. The author of the feature branch can review and merge the translations in and now any newly added features with always have all the static content in all the configured locales.
+### Content Translations
 
-#### Backend
+**Problem**: Translating the UI based on user's language preference
```

Comment:
I think lot of the following can be merged into one section and we can remove some duplication while we are at it. This way problem can become a good 3-4 line problem 
The solution should have a 3-4 line summary in terms of what philosophy we embraced and they bullet points for emails, ui, content etc